htdocs/login.bml

```diff
@@ -204,7 +204,7 @@ _c?>
             $body .= "<div class='login-create-account'>\n";
             $body .= "<hr class='hr' />\n";
             $body .= "<h4>$ML{'.createaccount.header'}</h4>\n";
-            $body .= "<form action='<?siteroot?>/create' method='get'><input type='submit' value='$ML{'.createaccount.button'}' class='submit' /></form>\n";
+            $body .= "<form action='/create' method='get'><input type='submit' value='$ML{'.createaccount.button'}' class='submit' /></form>\n";
```

Comment:
I always feel vaguely uneasy about unanchored things, but in this case it's just a GET request so it's fine but also I think it's actually fine. We'd never show this login page on someone's account domain...